src/main/docs/guide/views/templates/jte.adoc

```diff
@@ -0,0 +1,37 @@
+Micronaut includes api:views.jte.JteViewsRenderer[] which uses
+the https://jte.gg[Jte] Java-based template engine.
+
+Add the `micronaut-views-jte` dependency to your classpath.
+
+dependency:io.micronaut.views:micronaut-views-jte[group="io.micronaut.views"]
+
+The example shown in the <<views, Views>> section, could be rendered with the following Jte template:
+
+[source,html]
+.src/main/resources/views/home.jte
+----
+include::{testsviewsJte}/resources/views/home.jte[]
+----
+
+=== Compiling Templates
+
+Jte templates may be precompiled at build time. This can be done by a Gradle plugin. If not precompiled, the application will need a JDK so it can compile templates at runtime.
```

Comment:
There is a maven plugin, but I personally am unfamiliar with maven so didn't try using it.

Reply:
Is it important to include Maven support as well?

Reply:
Yes, we are build framework agnostic. If a feature is only possible with Gradle. Then, no problem we can have a feature only for Gradle. However, It would be good to investigate it. 

Ideally, after we merge this feature I would like to create a [guide](https://githu.com/micronaut-projects/micronaut-guides) which shows how to use JTE with Gradle and Maven.

Reply:
I made a Maven sample with some judicious copy-pasting, so I've added the Maven plugin to the docs.